dev-util/hdocs/hdocs-0.5.5.0-r1.ebuild

```diff
@@ -26,6 +26,7 @@ RDEPEND=">=dev-haskell/aeson-0.7.0:=[profile?]
 	>=dev-haskell/network-2.4.0:=[profile?]
 	>=dev-haskell/text-1.1.0:=[profile?]
 	>=dev-lang/ghc-8.4.3:=
+	<dev-lang/ghc-9:=
```

Comment:
So with this, I believe instead of giving us the intended result of basically masking hdocs and hsdev if you're on ghc-9, it makes Portage spend a bit longer on dependency resolution. At worst it can give the user conflicts which need working through. I think with these packages they need to be masked. I've already masked hsdev, this one can be too.